src/m365/spo/commands/applicationcustomizer/applicationcustomizer-remove.ts

```diff
@@ -0,0 +1,212 @@
+import { Cli } from '../../../../cli/Cli';
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { validation } from '../../../../utils/validation';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import { spo } from '../../../../utils/spo';
+import { CustomAction } from '../../commands/customaction/customaction';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  title?: string;
+  id?: string;
+  clientSideComponentId?: string;
+  scope?: string;
+  confirm?: boolean;
+}
+
+class SpoApplicationCustomizerRemoveCommand extends SpoCommand {
+  private static readonly scopes: string[] = ['Site', 'Web', 'All'];
+
+  public get name(): string {
+    return commands.APPLICATIONCUSTOMIZER_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Remove an application customizer from a site';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-u, --webUrl <webUrl>'
+      },
+      {
+        option: '-t, --title [title]'
+      },
+      {
+        option: '-i, --id [id]'
+      },
+      {
+        option: '-c, --clientSideComponentId [clientSideComponentId]'
+      },
+      {
+        option: '-s, --scope [scope]', autocomplete: SpoApplicationCustomizerRemoveCommand.scopes
+      },
+      {
+        option: '--confirm'
+      }
+    );
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        clientSideComponentId: typeof args.options.clientSideComponentId !== 'undefined',
+        id: typeof args.options.id !== 'undefined',
+        title: typeof args.options.title !== 'undefined',
+        scope: typeof args.options.scope !== 'undefined'
+      });
+    });
+  }
+
+  #initValidators(): void {
+    this.validators.push(this.validateUrl);
```

Comment:
You should push just one validator. And also: to be consistent with the rest of the codebase, let's not use class methods here, let's just write the validation code inside this function, like you see is done in other commands. Please compare with `applicationcustomizer-get.ts`.